### PR TITLE
Inner color branch

### PR DESCRIPTION
**Pull request by Maryom**
Hey,

Regarding this point:
> - Make sure the Example project compiles and works fine in the Simulator

I tried it, it always shows black color!

It worked perfectly before adding the changes you asked for. Any suggestion? Did I forget something?

### CODE REVIEW

File: Source/AIFlatSwitch.swift [hunk 1]
```diff
@@ -52,6 +52,15 @@ import UIKit
         }
     }
     
+    /**
+     Color for the inner circle.
+     */
+    @IBInspectable open var backgroundLayerColor: UIColor = UIColor.clear {
+        didSet {
+            self.backgroundLayer.backgroundColor = backgroundLayerColor.cgColor
```
Comment (cocoatoucher) [inline Source/AIFlatSwitch.swift]: This should be self.backgroundLayer.fillColor instead

File: Source/AIFlatSwitch.swift [hunk 2]
```diff
@@ -221,6 +241,9 @@ import UIKit
         }
         
         // Setup layers
+        self.layer.addSublayer(backgroundLayer)
+        backgroundLayer.backgroundColor = backgroundLayerColor.cgColor
```
Comment (cocoatoucher) [inline Source/AIFlatSwitch.swift]: Same here, it should be backgroundLayer.fillColor, not backgroundColor

**Review by cocoatoucher — CHANGES_REQUESTED** [review]

**Comment by cocoatoucher** [comment]
Hey, there is just a small error, you should use fillColor on layers instead of backgroundColor
Thanks very much for all the changes you made :)

**Comment by Maryom** [comment]
Most welcome :) I will fix them ASAP

**Comment by cocoatoucher** [comment]
Thank you very much!

**Comment by Maryom** [comment]
Most welcome

**Pull request merged**